internal/controller/ksanstorage_controller.go

```diff
@@ -27,6 +27,8 @@ import (
 	ksanv1alpha1 "openshift/ksan-operator/api/v1alpha1"
 )
 
+const operatorNamespace = "openshift-ksan-operator"
```

Comment:
We shouldn't hardcode this, we should use the environment to pull what namespace the operator is running in. LVMS does it like this:
https://github.com/openshift/lvm-operator/blob/main/internal/cluster/namespace.go